src/main/scala/com/ing/wbaa/rokku/sts/service/UserTokenDbService.scala

```diff
@@ -13,9 +13,9 @@ trait UserTokenDbService extends LazyLogging with TokenGeneration {
 
   implicit protected[this] def executionContext: ExecutionContext
 
-  protected[this] def getAwsCredential(userName: UserName): Future[Option[AwsCredential]]
+  protected[this] def getAwsCredential(userName: UserName): Future[(Option[AwsCredential], Boolean)]
 
-  protected[this] def getUserSecretKeyAndIsNPA(awsAccessKey: AwsAccessKey): Future[Option[(UserName, AwsSecretKey, Boolean, Set[UserGroup])]]
+  protected[this] def getUserSecretKeyAndIsNPA(awsAccessKey: AwsAccessKey): Future[Option[(UserName, AwsSecretKey, Boolean, Boolean, Set[UserGroup])]]
```

Comment:
the same problem with method name :)
because we have so many returned parameters maybe we can put all of the in a case class and 
```suggestion
  protected[this] def getUser(awsAccessKey: AwsAccessKey): Future[Option[User]]
```

Reply:
I'll wrap Bools in case classes, indeed becoming unreadable